frontend/src/stores/userStore.ts

```diff
@@ -12,13 +12,14 @@ export const useUserStore = defineStore('users', {
 		errors: [],
```

Comment:
You can type this:
`errors: [] as String[]`

Reply:
I also think that we need to figure something out that these are reset on every sending of the form, and at the change of a page.
Because now, if you do something "wrong" in the form, and then do the right thing, the errors don't disappear..

Reply:
this should work:
```ts
{
   errors: [] as String[]
}
```